README.md

```diff
@@ -380,7 +402,7 @@ method takes two arguments:
 ### Profunctor
 
 A value that implements the Profunctor specification must also implement
-the Functor specification.
+the Functor and Contravariant Functor specifications.
```

Comment:
I'm in two minds about including this here.

In some other typed languages we'd have an issue where the variance of the type arguments of contravariant and profunctors don't line up. We don't have this issue with a regular covariant functor and profunctor because both last type arguments are covariant (we just have to partially apply the type constructor to take the kind of profunctor from `* -> * -> *` to `* -> *`).

The other half of my mind says this is probably being pedantic as we're not bound by such a type system so it's not really a problem.